<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DreamNotebook</class>
 <widget class="QMainWindow" name="DreamNotebook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>DreamNotebook</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>70</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="documentMode">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="tabMode">
         <attribute name="title">
          <string>Режим</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QToolButton" name="buttDreams">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>Дневник</string>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/icon/icons/dreambook.png</normaloff>:/icon/icons/dreambook.png</iconset>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <property name="autoExclusive">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="buttTrash">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>Корзина</string>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/icon/icons/trash.png</normaloff>:/icon/icons/trash.png</iconset>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="autoExclusive">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tabDel">
         <attribute name="title">
          <string>Удаление</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QCheckBox" name="checkDel">
            <property name="text">
             <string>Удалить</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkDestroy">
            <property name="text">
             <string>Удалять, минуя корзину</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="newDream">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Новый</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icon/icons/create.png</normaloff>:/icon/icons/create.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="shortcut">
         <string>Ctrl+N</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="spacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>70</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTreeView" name="dreamTree"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Информация</string>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/icon/icons/documentation.png</normaloff>:/icon/icons/documentation.png</iconset>
    </property>
    <addaction name="aboutProg"/>
    <addaction name="source"/>
    <addaction name="separator"/>
    <addaction name="aboutQt"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="aboutProg">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icon/icons/aboutProg.png</normaloff>:/icon/icons/aboutProg.png</iconset>
   </property>
   <property name="text">
    <string>О програмем</string>
   </property>
  </action>
  <action name="source">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icon/icons/C++.png</normaloff>:/icon/icons/C++.png</iconset>
   </property>
   <property name="text">
    <string>Исходники</string>
   </property>
  </action>
  <action name="aboutQt">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icon/icons/aboutQt.png</normaloff>:/icon/icons/aboutQt.png</iconset>
   </property>
   <property name="text">
    <string>О Qt</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
